### PR TITLE
Feature accessibility

### PR DESCRIPTION
**Pull request by SirriCelles**
## Pair Programming: Feature accessibility
- Check for page title -> No issue found.
-  Image text alternative -> change from `object ` to `img`. [link1](https://github.com/SirriCelles/portfolio/blob/b666748e25835c99700cf2b1d88709937c91a451/index.html#L51-L93)  and [link 2](https://github.com/SirriCelles/portfolio/blob/b666748e25835c99700cf2b1d88709937c91a451/index.html#L212-L251).

- Text Headings  Checks -> Passed
- Color Contrast -> Passed
- Page Resize check -> Passed
- Interaction checks passed -> Passed
- Moving Content Checks -> Passed
- Multimedia Checks -> Passed
- Basic page layout setup -> Passed

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,288 +1,388 @@
...
+              </a>
+            </li>
           </ul>
+        </div>
+        <button class="tonic__btn" type="button">Get my resume</button>
       </div>
-      <button class="tonic__btn" type="button">
-      Get my resume
-      </button>
-    </div>
```
Comment (KDlamini) [inline index.html:207]: The `About section` is not accessible using the keyboard tab focus. The focus jumps from the frameworks section to the contact form. Kindly consider using the `tabindex` attributes on the elements to make them accessible using the keyboard. You can read more about [Keyboard Accessibility](https://webaim.org/techniques/keyboard/tabindex) using tabindex.

image

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @SirriCelles, @jahongiry,  @gasangw 🙋‍♀️,  

Great job so far!
You have done a great job implementing accessibility for your project. 🎊 
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  Correct PR title and description
✔️  Correct use of Gitflow 
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (abdulhamiid) [reply index.html:207]: @KDlamini,
I tried to verify everything in that area using the ``` tab ``` button, and everything works fine.

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @SirriCelles,

MissionCompleteGIF (2)

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**